src/__tests__/suggestions.ts

```diff
@@ -187,39 +188,41 @@ test('escapes regular expressions in suggestion', () => {
   )
 
   expect(
-    getSuggestedQuery(
-      container.querySelector('img'),
+    (getSuggestedQuery(
+      container.querySelector('img') as HTMLImageElement,
```

Comment:
question: Doesn't dom.d.ts infer the element type? Or does this only work for document.querySelector?

Reply:
the problem is that querySelector will return null or the element

Reply:
Then this is a job for the non-null assertion operator `!`.

Reply:
we have a rule `@typescript-eslint/no-non-null-assertion`, we can disable it for the project, what do you think ?

Reply:
> we have a rule @typescript-eslint/no-non-null-assertion, we can disable it for the project, what do you think ?

I'd disable it. We're already violating the rationale by type-casting anyway.